### PR TITLE
Add documentation for the TreeBehavior recoverOrder

### PR DESCRIPTION
**Pull request by berrygoudswaard**
This is the documentation for the the following pull request: cakephp/cakephp#7411

Note: This pull request is not accepted yet, so the documentation is not relevant yet.

### CODE REVIEW

File: en/orm/behaviors/tree.rst [hunk 1]
```diff
@@ -214,6 +214,20 @@ as the scope::
         return $query->where(['country_name' => $country]);
     });
 
+Recovering with custom sort field
+=================================
+
+By default the recover method sorts the items using the primary key. This works great
+if this is a numeric (auto increment) column, but can lead to weird results if you
+use UUID's.
```
Comment (ravage84) [inline en/orm/behaviors/tree.rst]: UUIDs

File: en/orm/behaviors/tree.rst [hunk 2]
```diff
@@ -214,6 +214,20 @@ as the scope::
         return $query->where(['country_name' => $country]);
     });
 
+Recovering with custom sort field
+=================================
+
+By default the recover method sorts the items using the primary key. This works great
```
Comment (ravage84) [inline en/orm/behaviors/tree.rst]: "By default, ..."

Comment (ravage84) [inline en/orm/behaviors/tree.rst]: ``recover() ``, may be?

**Comment by berrygoudswaard** [comment]
@ravage84 Thanks for the feedback :+1:

**Pull request merged**